tools/bbbsupervisor/bbbsupervisor.go

```diff
@@ -105,6 +126,64 @@ func (lf logFollower) follow() {
 	errWriter.Write([]byte(fmt.Sprintf("command %v unexpectedly exited", fullCmd)))
 }
 
+// follow implements follower interface by following Prometheus server and querying for values forever.
+func (pl prometheusFollower) follow() {
+	for {
+		fmt.Printf("prometheusFollower querying for %q\n", pl.expression)
+		pl.query()
+		time.Sleep(30000 * time.Millisecond)
+	}
+}
+
+// query queries the Prometheus server once.
+func (pl prometheusFollower) query() {
```

Comment:
`pl` should be `pf`?

Reply:
Oops, good catch. (This used to be called `prometheusLogger` or something; I renamed the type but forgot the method receiver.)

Fixed here and elsewhere.